Lombiq.Tests.UI/Models/FailureDumpItem.cs

```diff
@@ -8,14 +10,17 @@ public class FailureDumpItem : IFailureDumpItem
 {
     private readonly Func<Task<Stream>> _getStream;
     private readonly Action _dispose;
+    private readonly IEnumerable<Type> _inCaseOf;
```

Comment:
We don't really need this anymore (and in `FailureDumpItemGeneric` and else), no?